google-ads-examples/src/main/java/com/google/ads/googleads/examples/remarketing/AddListBasedUserList.java

```diff
@@ -0,0 +1,152 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.ads.googleads.examples.remarketing;
+
+import com.beust.jcommander.Parameter;
+import com.google.ads.googleads.examples.utils.ArgumentNames;
+import com.google.ads.googleads.examples.utils.CodeSampleParams;
+import com.google.ads.googleads.lib.GoogleAdsClient;
+import com.google.ads.googleads.v1.utils.ResourceNames;
+import com.google.ads.googleads.v2.common.LogicalUserListInfo;
+import com.google.ads.googleads.v2.common.LogicalUserListOperandInfo;
+import com.google.ads.googleads.v2.common.UserListLogicalRuleInfo;
+import com.google.ads.googleads.v2.enums.UserListLogicalRuleOperatorEnum.UserListLogicalRuleOperator;
+import com.google.ads.googleads.v2.errors.GoogleAdsError;
+import com.google.ads.googleads.v2.errors.GoogleAdsException;
+import com.google.ads.googleads.v2.resources.UserList;
+import com.google.ads.googleads.v2.services.MutateUserListsResponse;
+import com.google.ads.googleads.v2.services.UserListOperation;
+import com.google.ads.googleads.v2.services.UserListServiceClient;
+import com.google.common.collect.ImmutableList;
+import com.google.protobuf.StringValue;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Creates a combination user list containing users that are present on any one of the provided user
+ * lists.
+ */
+public class AddListBasedUserList {
```

Comment:
Could we have a more descriptive name for this example perhaps? I find AddListBasedUserList confusing, partly due to the repitition of "list". How about AddUnionUserList, AddCombinationUserList, MergeUserLists?

Reply:
Another option might be AddLogicalUserList if we don't expect any other examples with the same name.

Reply:
I like AddUnionUser list. Given that there are several other user list examples, I want to be specific about what this example does as to avoid confusion.